pages/dkp/kommander/1.4/clusters/attach-cluster/index.md

```diff
@@ -82,32 +86,86 @@ users:
 EOF
 ```
 
-This produces a file in your current working directory called `kommander-cluster-admin-config`. The contents of this file are used in Kommander to attach the cluster.
+This procedure produces a file in your current working directory called, `kommander-cluster-admin-config`. The contents of this file are used in Kommander to attach the cluster.
 
-Before importing this configuration, you can verify it is functional by running the following command:
+Before importing this configuration, you can verify that it is functional by running the following command:
 
 ```shell
 kubectl --kubeconfig $(pwd)/kommander-cluster-admin-config get all --all-namespaces
 ```
 
 ### Attaching a Cluster
 
-Using the **Add Cluster** option, you can attach an existing Kubernetes or Konvoy cluster directly to Kommander. You can access the multi-cluster management and monitoring benefits Kommander provides while keeping your existing cluster on its current provider and infrastructure.
+Using the **Add Cluster** option, you can attach an existing Kubernetes or Konvoy cluster directly to Kommander. Attaching a cluster allows you to access the multi-cluster management and monitoring benefits that Kommander provides, while keeping your existing cluster on its current provider and infrastructure.
+
+You have these security options when attaching a cluster:
+
+- Attach a cluster with no additional security restrictions
```

Comment:
Should not it be `with no additional networking restrictions` instead of `security` ?

Reply:
That is the difference from one to another

Reply:
You are right - good catch!